DB/projects.json

```
@@ -5772,3 +5772,28 @@
     ]
   }
 ]
+
+{
+  "github_username": "Aryainguz",  
+  "Social_media": {
+    "gitHub": "https://github.com/Aryainguz",
+    "LinkedIn": "https://www.linkedin.com/in/aryan-inguz-b73150254/",
+    "Instagram": "https://instagram.com/aryaninguz?igshid=NTc4MTIwNjQ2YQ==",
+    "YouTube": "https://www.google.co.in/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjLn5XUzsaAAxVkZmwGHfE4BmcQFnoECB8QAQ&url=https%3A%2F%2Fwww.youtube.com%2F%40thexitingway%2Fchannels&usg=AOvVaw2xudaOlyH62PUapOqTE736&opi=89978449",
+    "Twitter": "https://www.google.co.in/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjnu8nhzsaAAxUod2wGHdlbCc0QFnoECB4QAQ&url=https%3A%2F%2Ftwitter.com%2Faryaninguz369&usg=AOvVaw0SbGgB5S-hUgVQz4o5yRup&opi=89978449"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/Aryainguz/Euphoria-Check-PERMA-Meter-Express",
+      "title": "EuphoriaCheck - A Modern PERMA Meter",
+      "description": "A powerful tool to help you assess and improve your well-being! This web application, built using Express and Node.js, measures your happiness and life satisfaction based on the PERMA model of well-being. With personalized insights, users gain a comprehensive breakdown of their PERMA scores, identifying areas for personal growth and well-being enhancement.Users recieves a personalized note on email that helps them set their targets and improve on specific score so they keep on evolving Key Features: 📊 Happiness Assessment: 📈 Personalized Insights: 🎯 Goal Setting",
+      "tech": ["ExpressJS", "NodeJS"]
+    },
+    {
+      "link": "https://github.com/Aryainguz/Email-Newsletter-Subscription-Web-App-Using-ExpressJS",
+      "title": "Email-Newsletter-Subscription-Web-App-Using-ExpressJS",
+      "description": "Email Marketing is an important aspect of promotions of any kind, so here's a email newsletter subscription web app i made using expressjs to get subscribers for my email marketing and newsletters . Using Express JS and Mailchimp's API do use it for your personal project and share you insights with me, Aryan",
+      "tech": ["ExpressJS", "REST API"]
```

convert to lowercase 
```suggestion
      "tech": ["expressjs", "api"]
```